locales/en/app.ftl

```diff
@@ -865,6 +863,8 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 -brand-mozilla = Mozilla
```

Comment:
Interesting. We have a `-brand-mozilla` (here) and `-brand-Mozilla` (L10).
Not sure if that is fallout from v1 and v2 or not.
Looks like we also have `-brand-firefox` and `-brand-name` and both are "Firefox".

```sh
git grep "^-brand-" locales/en/app.ftl | sort -f

locales/en/app.ftl:-brand-firefox = Firefox
locales/en/app.ftl:-brand-fpn = Firefox Private Network
locales/en/app.ftl:-brand-fx-account = Firefox account
locales/en/app.ftl:-brand-fx-monitor = Firefox Monitor
locales/en/app.ftl:-brand-fxa = Firefox Account
locales/en/app.ftl:-brand-github = GitHub
locales/en/app.ftl:-brand-HIBP = Have I Been Pwned
locales/en/app.ftl:-brand-lockwise = Firefox Lockwise
locales/en/app.ftl:-brand-Mozilla = Mozilla
locales/en/app.ftl:-brand-mozilla = Mozilla
locales/en/app.ftl:-brand-mozilla-foundation = Mozilla Foundation
locales/en/app.ftl:-brand-mozilla-vpn = Mozilla VPN
locales/en/app.ftl:-brand-name = Firefox
locales/en/app.ftl:-brand-pocket = Pocket
locales/en/app.ftl:-brand-relay = Firefox Relay
locales/en/app.ftl:-brand-send = Firefox Send
```

Or if you want sorted by line number, this does the trick and lets you see which is v1 vs v2:
```sh
git grep -n "^-brand-" locales/en/app.ftl | cat

# Monitor V1
locales/en/app.ftl:9:-brand-name = Firefox
locales/en/app.ftl:10:-brand-Mozilla = Mozilla
locales/en/app.ftl:11:-brand-HIBP = Have I Been Pwned
locales/en/app.ftl:12:-brand-fxa = Firefox Account
locales/en/app.ftl:13:-brand-pocket = Pocket
locales/en/app.ftl:14:-brand-lockwise = Firefox Lockwise
locales/en/app.ftl:15:-brand-send = Firefox Send
locales/en/app.ftl:16:-brand-fpn = Firefox Private Network

# Monitor V2
locales/en/app.ftl:861:-brand-firefox = Firefox
locales/en/app.ftl:862:-brand-fx-monitor = Firefox Monitor
locales/en/app.ftl:863:-brand-mozilla = Mozilla
locales/en/app.ftl:864:-brand-mozilla-foundation = Mozilla Foundation
locales/en/app.ftl:865:-brand-github = GitHub
locales/en/app.ftl:866:-brand-mozilla-vpn = Mozilla VPN
locales/en/app.ftl:867:-brand-relay = Firefox Relay
locales/en/app.ftl:873:-brand-fx-account = Firefox account
```

Not sure if the plan is to remove all the v1 strings from the .ftl + Pontoon once v2 is live or not, just thought it was mildly interesting.

Reply:
yeah, these will all be cleaned up post-launch, but I appreciate you keeping tabs. `-brand-name` was an unfortunate naming convention used in the legacy site that should cease to exist in the new site.